src/localstorage.js

```diff
@@ -0,0 +1,25 @@
+export const name = document.querySelector('.contact__name');
+
+export function storageAvailable(type) {
+  let storage;
+  try {
+    storage = window[type];
+    const x = '__storage_test__';
+    storage.setItem(x, x);
+    storage.removeItem(x);
+    return true;
+  } catch (e) {
+    return e instanceof DOMException && (
+    // everything except Firefox
+      e.code === 22
+            // Firefox
+            || e.code === 1014
+            // test name field too, because code might not be present
+            // everything except Firefox
+            || e.name === 'QuotaExceededError'
+            // Firefox
+            || e.name === 'NS_ERROR_DOM_QUOTA_REACHED')
+            // acknowledge QuotaExceededError only if there's something already stored
+            && (storage && storage.length !== 0);
+  }
+}
```

Comment:
- [ ] You are almost there! But we still need to use `localStorage` to meet the project requirements: 

You should implement the following interactions:

- When the user changes the content of any input field, the data is saved to the local storage.

- When the user loads the page, if there is any data in the local storage the input fields are pre-filled with this data.

You should use the following data model:

- You must create a single JavaScript object with all the data from the entire form and save it in local storage. 
- Do not create one local storage entry per input field.